src/components/v5/shared/Filters/Filters.tsx

```diff
@@ -0,0 +1,411 @@
+import { DeepPartial } from 'utility-types';
```

Comment:
Is putting this component here the right choice? It's not used by anything else other than Balance and Funds table, and we have alternative implementation of table filters in members page, activity feed, etc. Maybe it's better to keep it specific to the balance/funds tables?